From 1c65432ddac7cc744fd0fe1e8a5f2ebdfc0041d6 Mon Sep 17 00:00:00 2001
From: Sven Obser <sven.obser@backes-srt.de>
Date: Mon, 2 Oct 2017 14:25:06 +0200
Subject: [PATCH 04/13] Fixes Issue #9: Invalid types in LogRedeemPLT event

---
 contracts/PLToken.sol | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/contracts/PLToken.sol b/contracts/PLToken.sol
index b71bd3d..d8239f4 100644
--- a/contracts/PLToken.sol
+++ b/contracts/PLToken.sol
@@ -202,7 +202,7 @@ contract PLToken is StandardToken {
 
 
     /// @dev Redeems PLTs and records the LEO address of the sender
-    function redeemTokens(bytes32 leoAddress)
+    function redeemTokens(string leoAddress)
     external
     isRedeeming
     {
-- 
2.7.4.windows.1

