From 26a4b90b7dbe1bfbed53a0b9dedfe5d5a86bb76e Mon Sep 17 00:00:00 2001
From: Sven Obser <sven.obser@backes-srt.de>
Date: Mon, 2 Oct 2017 17:07:46 +0200
Subject: [PATCH 05/13] Fixes Issue #3: Ensure admin accounts are different

---
 contracts/PLToken.sol | 1 +
 1 file changed, 1 insertion(+)

diff --git a/contracts/PLToken.sol b/contracts/PLToken.sol
index d8239f4..cb26bea 100644
--- a/contracts/PLToken.sol
+++ b/contracts/PLToken.sol
@@ -107,6 +107,7 @@ contract PLToken is StandardToken {
         require(_exchangeRateChangesBlock <= _fundingEndBlock); // And the end of the fundraising should happen after the exchange rate change
         require (_admin1 != 0x0); // admin1 address must be set
         require (_admin2 != 0x0); // admin2 address must be set
+        require (_admin1 != _admin2); // Ensure that admin accounts are different
 
         // Contract state
         state = ContractState.Fundraising;
-- 
2.7.4.windows.1

