From 8b83b49cfec7a0002df7207b8bd34b90084823f2 Mon Sep 17 00:00:00 2001
From: Sven Obser <sven.obser@backes-srt.de>
Date: Mon, 2 Oct 2017 13:33:18 +0200
Subject: [PATCH 10/13] Remove commented code.

---
 contracts/PLToken.sol | 1 -
 1 file changed, 1 deletion(-)

diff --git a/contracts/PLToken.sol b/contracts/PLToken.sol
index 53dd843..9b19697 100644
--- a/contracts/PLToken.sol
+++ b/contracts/PLToken.sol
@@ -215,7 +215,6 @@ contract PLToken is StandardToken {
         require(netVal >= TOKEN_MIN); // At least TOKEN_MIN tokens have to be redeemed
 
         // Burn Tokens on redemption
-        //if (!super.transfer(0x0, netVal)) throw;
         require(super.transfer(0x0, netVal));
         // Log the redeeming of this tokens
         LogRedeemPLT(msg.sender, netVal, leoAddress);
-- 
2.7.4.windows.1

