From 64e67b9142d8b01b8c946cbb23c37c6103e00423 Mon Sep 17 00:00:00 2001
From: Sven Obser <sven.obser@backes-srt.de>
Date: Wed, 4 Oct 2017 09:35:41 +0200
Subject: [PATCH 12/13] Made version constant.

---
 contracts/PLToken.sol | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/contracts/PLToken.sol b/contracts/PLToken.sol
index 69e7e88..cde7351 100644
--- a/contracts/PLToken.sol
+++ b/contracts/PLToken.sol
@@ -13,7 +13,7 @@ contract PLToken is StandardToken {
     string public constant name = "Pre-Leo Token";
     string public constant symbol = "PLT";
     uint256 public constant decimals = 18;
-    string public version = "0.9";
+    string public constant version = "0.9";
 
     // Fundraising parameters
     enum ContractState { Fundraising, Finalized, Redeeming, Paused }
-- 
2.7.4.windows.1

