From ac40a3c7fe7fd179fb5b78a12b20dcdacf9dd0d8 Mon Sep 17 00:00:00 2001
From: Sven Obser <sven.obser@backes-srt.de>
Date: Wed, 4 Oct 2017 11:52:37 +0200
Subject: [PATCH 13/13] Fixes Issue #11: One admin can replay last executed
 operation without approval

---
 contracts/PLToken.sol | 4 ++++
 1 file changed, 4 insertions(+)

diff --git a/contracts/PLToken.sol b/contracts/PLToken.sol
index cde7351..c63d6ec 100644
--- a/contracts/PLToken.sol
+++ b/contracts/PLToken.sol
@@ -86,6 +86,10 @@ contract PLToken is StandardToken {
         if ((multiSigHashes[admin1]) == (multiSigHashes[admin2])) {
             // if yes, both admins agreed - continue.
             _;
+
+            // Reset hashes after successful execution
+            multiSigHashes[admin1] = 0x0;
+            multiSigHashes[admin2] = 0x0;
         } else {
             // if not (yet), return.
             return;
-- 
2.7.4.windows.1

